src/main/java/de/teamlapen/vampirism/player/vampire/skills/VampireSkills.java

```diff
@@ -161,7 +162,7 @@ protected void onEnabled(IVampirePlayer player) {
             @Override
             public ITextComponent getDescription() {
                 if (Balance.vps.DISABLE_AVOIDED_BY_CREEPERS) {
-                    return TextFormatting.RED + "Disabled by admin" + TextFormatting.RESET;
+                    return new TextComponentString(TextFormatting.RED + "Disabled by admin" + TextFormatting.RESET);
```

Comment:
This can now be down slightly more elegantly by using #applyTextStyle (or similar)